test/util/keyring.ts

```diff
@@ -0,0 +1,137 @@
+import type { PolkadotSigner, SS58String } from "polkadot-api";
+import { fromHex } from "@polkadot-api/utils";
+import { ed25519 } from "@noble/curves/ed25519";
+import { sha256 } from "@noble/hashes/sha256";
+import { Keyring } from "@polkadot/api";
+import { waitReady } from "@polkadot/wasm-crypto";
+import { blake2b } from "@noble/hashes/blake2b";
+import { secp256k1 } from "@noble/curves/secp256k1";
+import { getPolkadotSigner } from "polkadot-api/signer";
```

Comment:
My TypeScript shows me the error `Module '"polkadot-api/signer"' has no exported member 'getPolkadotSigner'.`. Might just be something on my end, but I wanted to point it out anyway.

Reply:
After `bun install` you may have to restart your TS lang server. I had that error but it goes away after you give it a poke:

<img width="1082" alt="image" src="https://github.com/Moonsong-Labs/storage-hub/assets/79199034/be26fa66-3ce3-4882-a9dc-608195463c14">

Reply:
to restart ts lang server its in the cmd palette:
<img width="856" alt="image" src="https://github.com/Moonsong-Labs/storage-hub/assets/79199034/539e79b1-d80f-4adf-9ba6-50aef5a63d0f">